### DISCUSSION TITLE
Basic keyboard shortcuts for saving records

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13304] [Feature Requests]
I know about https://github.com/directus/directus/discussions/8407

However a few really basic shortcuts for starters like `CTRL+ENTER` for saving and go back to index, and another one for saving and remain editing I think will go a long way.

This isn't VIM, its users don't need a million keyboard shortcuts!

**Comment by benhaynes** [comment]
Let's keep all keyboard shortcuts consolidated in the one ticket though, it makes it much easier to track all potential options. Also, in my opinion, command-enter is for submitting one off fields, like comments, not entire forms. I'd prefer to see a command-s instead.

**Reply by sxsws** [reply]
i think ctrl+s is an application shortcut and i don't like it when websites mess with those!

**Reply by benhaynes** [reply]
I agree for websites, but disagree for formal WebApps.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
